src/pages/components/progress-indicator/usage.mdx

```diff
@@ -45,41 +48,192 @@ set of steps, guiding toward the completion of a specified process.
-#### Logical progression
+### Alignment
 
-Display the steps in order from left to right. Indicate to the user that they
-are performing a multistep process, and show the direction of movement. Allow
-the user to return to a previous step to review their data submission.
+Progress indicators can be in vertical or horizontal position depending on the
+use case and the structure of the UI. When possible, arrange the progress
+indicator vertically for easier reading.
 
-#### Indicate the current step
+<Row>
+<Column colLg={8}>
+
+![Example of a progress indicator used with vertical alignment](images/progress-indicator-usage-4.png)
+
+</Column>
+</Row>
 
-Keeping the user informed of where they currently are within the process or task
-at hand will give them a sense of control. This helps the user to know where
-they are in relation to where they have been, and what sections are to follow.
-Clear labels should accompany the progress indicator to indicate what the user
-will accomplish within each step. Keep labels between one to two words.
+### Placement
 
-#### Validation
+The progress indicator component is often used in
+[forms](https://carbondesignsystem.com/components/form/usage). Forms can be
+placed on a full page, in a modal, or in a side panel.
 
-Use validation to confirm that a previous step has been completed. If the user
-cannot proceed onto another step without first completing a task, use an
-[Inline Notification](/components/notification/code#inline-notification) to
-inform them.
+IMAGE TK: Get high res image. Likely needs design work first.
 
 <Row>
 <Column colLg={8}>
 
-![Example of the progress indicator in a payment UI](images/progress-indicator-usage-1.png)
+![Examples of vertical and horizontal progress indicators in products](images/progress-indicator-usage-3.png)
 
 </Column>
 </Row>
 
+## Content
+
+### Main elements
+
+#### Label
+
+The label should communicate what the user will accomplish in each step in one
+or two words. One option is to clearly indicate the action of the step by using
+the {verb} + {noun} content formula. For example, "Configure IdP" or "Define
+endpoints." A label my use one word if it is universally understood, such as
+"Cart," "Shipping," or "Payment." Avoid vague terms like "Processing." You may
+also number each step to make the progression more obvious.
+
+#### Helper text
+
+Use the helper text to indicate that a step is optional or that a step is in an
+error state.
+
+### Overflow content
+
+#### Label
+
+When there is not enough space, consider rewording the label or truncate the
+label text with an ellipsis and provide a tooltip to convey additional
+information.
+
+#### Helper text
+
+Long helper text may wrap to a second line, and this is preferable to
+truncation. Wrap the helper text beneath the label so that both maintain left
+alignment.
+
+<Row>
+<Column colLg={8}>
+
+![Example of progress indicator with overflow content](images/progress-indicator-usage-6.png)
+
+</Column>
+</Row>
+
```

Comment:
We usually add a caption for an image. It could be something like "Example shows text wraps to a second line in a long helper text."